var/spack/repos/builtin/packages/magma/package.py

```diff
@@ -210,7 +210,9 @@ def test_c(self):
         with working_dir(test_dir):
             pkg_config_path = self.prefix.lib.pkgconfig
             with spack.util.environment.set_env(PKG_CONFIG_PATH=pkg_config_path):
-                make("c")
+                make = self.spec["gmake"].command
+                CC = "hipcc" if self.spec.satisfies("+rocm") else self.compiler.cc
```

Comment:
It's unfortunate that `self.compiler.cc` doesn't "good" compiler for `rocm` specs.